extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -266,6 +268,43 @@
   margin-right: 32px !important;
 }
 
+.i-amphtml-story-bookend-cta-link-wrapper {
+  display: flex !important;
+  flex-wrap: wrap !important;
+  box-sizing: border-box !important;
+  color: inherit !important;
+  overflow: hidden !important;
+  flex: 1 0 320px !important;
+  justify-content: space-between !important;
+  margin: 24px 0 !important;
+  padding: 0 32px !important;
+}
+
+.i-amphtml-story-bookend-cta-link {
+  background-color: rgba(255, 255, 255, 0.15) !important;
+  border-radius: 100px !important;
+  overflow: hidden !important;
+  text-decoration: none !important;
+  width: 45% !important;
+  max-width: 170px !important;
+  cursor: pointer !important;
+  margin-bottom: 10px;
+}
+
+.i-amphtml-story-bookend-cta-link-text {
+  color: #FFFFFF !important;
+  font-family: 'Roboto', sans-serif !important;
+  text-transform: uppercase !important;
+  text-align: center !important;
+  font-size: 13px !important;
+  font-weight: bold !important;
+  line-height: 20px !important;
+  padding: 8% 10% 8% 10% !important;
```

Comment:
% padding are very confusing, and give us very little control over the actual design grid (right now buttons have a 42.38px height).
I'd suggest the following instead:

````
padding: 0 24px !important;
height: 40px !important;
line-height: 40px !important;
````

Reply:
makes sense, looks better!